src/test/java/net/sf/jabref/logic/importer/util/JSONEntryParserTest.java

```diff
@@ -39,7 +39,7 @@ public void testBibJSONConverter() {
                         + "\"id\": \"1563-5171\"},{\"type\": \"doi\",\"id\": \"10.1155/2014/217495\""
                 + "}],\"created_date\":\"2014-05-09T19:38:31Z\"}\"";
         JSONObject jo = new JSONObject(jsonString);
```

Comment:
Better variable names here, please. "jo" reminds me of the German slang word for ok;)